Snoop.Core/Data/TransientSettingsData.cs

```diff
@@ -40,6 +40,8 @@ public TransientSettingsData()
 
     public bool EnableDiagnostics { get; set; }
 
+    public bool StartMinimized { get; set; }
```

Comment:
There is no need to add this here as the new setting only affects Snoop.exe itself.